back/engines/commercial/analysis/app/lib/analysis/summarization_method/one_pass_llm.rb

```diff
@@ -77,17 +77,18 @@ def run(plan)
     def prompt(project, inputs_text)
       project_title = MultilocService.new.t(project.title_multiloc)
       @prompt = <<~GPT_PROMPT
-        At the end of this message is a list of form responses filled out by citizens in the context of an online participation project titled '#{project_title}'. The responses are separated by lines.
+        At the end of this message is a list of form responses filled out by respondents in the context of an online participation project titled '#{project_title}'. The responses are separated by lines.
+
+        Summarize what respondents have said. Keep it brief. Put the most emphasis on things that were mentioned most often.
+        The goal is for the reader to get an understanding of what respondents have been talking about, without having to read through it all. Focus more on the trends across the responses, than on individual responses.
         
-        Summarize what citizens have proposed in a few paragraphs. Be as complete as possible and put the most emphasis on things that were mentioned most often.
-        The goal is for the reader to get an understanding of what citizens have been talking about, without having to read through it all. Focus more on the trends across the ideas, than on individual ideas.
+        Do NOT start your summary with any kind of introduction, start right away with the content, because the reader already knows the project context and the fact that this is a summary.
```

Comment:
Just wondering, does "NOT" (capitalized) tend to work better than "not" (lowercase)?

Reply:
I didn't test, but this worked well.